cfgov/v1/wagtail_hooks.py

```diff
@@ -33,3 +36,42 @@ def check_request_site(page, request, serve_args, serve_kwargs):
         if isinstance(page, CFGOVPage):
             if not page.shared:
                 raise Http404
+
+
+PAGES_JSON = settings.PROJECT_ROOT.child('v1', '_generated') + '/pages.json'
+
+def update_key_pages_json(request, page):
+    page = page.specific
+    data = json.load(open(PAGES_JSON))
```

Comment:
@anselmbradford oh so it maintains a history of the pages that have been created, so on page delete i could just refer to the ID and remove it from the dict.. 
but now that i think about it, i could just add a delete flag so gulp knows to remove that path.. e.g
```
{
  "page_3": {
    "deleted": "true"
    "slug": "demo-purposes"
  }
}
```

@anselmbradford though i wonder if creating the page every-time might cause concurrency issues